services/processor/processor/processor.py

```diff
@@ -118,10 +118,11 @@ def start_processing(self):
         will trigger the `handlers/project_sync.py` since that one has the attribute
         REGISTER_EVENT_TYPE = ["create-project"]
         """
-        logging.info("Start enrolling for Ayon `shotgrid.event` Events...")
```

Comment:
debug?

Reply:
I replaced it with another one, but yeah, will set it as debug